sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_schema_registry_client.py

```diff
@@ -171,6 +171,34 @@ def get_schema(self, schema_id, **kwargs):
         response.raise_for_status()
         return _parse_response_schema(response)
 
+    def get_schema_by_version(self, group_name: str, name: str, version: int, **kwargs: Any) -> Schema:
+        """
+        Gets a registered schema within the specified schema group, given its name and version.
+
+        :param str group_name: Schema group under which schema should be registered.
```

Comment:
nit: this makes it sound a little like we are registering the schema here.

```suggestion
        :param str group_name: Name of schema group that contains the registered schema.
```